royalties-registry/contracts/providers/RoyaltiesProviderArtBlocks.sol

```diff
@@ -23,37 +23,72 @@ contract RoyaltiesProviderArtBlocks is IRoyaltiesProvider, Ownable {
         //gettign artist and additionalPayee royalty part
         (address artistAddress, address additionalPayee, uint256 additionalPayeePercentage, uint256 royaltyFeeByID) = artBlocks.getRoyaltyData(tokenId);
 
+        require(additionalPayeePercentage <= 100 && royaltyFeeByID <= 100, "wrong additionalPayeePercentage value");
```

Comment:
1.**require** check two conditions by value, comment only about one condition, maybe better do  two **require**?
require(additionalPayeePercentage <= 100, comment);
require(royaltyFeeByID <= 100, comment);
2.in comment "wrong additionalPayeePercentage value" instead of variable  additionalPayeePercentage maybe better use "additional payee  percentage"?

Reply:
fixed